components/dashboard/docker/logs/docker-logs-id.tsx

```diff
@@ -81,8 +82,8 @@ export const DockerLogsId: React.FC<Props> = ({ id, containerId }) => {
 				/>
 			</div>
 
-			<div className="w-full h-full bg-input rounded-lg p-2 ">
-				<div id={id} className="rounded-xl" />
+			<div className="w-full h-full bg-input rounded-lg p-2 bg-[#19191A]">
+				<div id={id} />
 			</div>
 		</div>
```

Comment:
Can you check please the colors of the terminal? currently in light mode it looks white and it have very low contrast

![Screenshot 2024-05-16 at 10 44 36 PM](https://github.com/Dokploy/dokploy/assets/47042324/0e79c1fc-6248-4e9a-a51d-60a6e452e6bd)
![Screenshot 2024-05-16 at 10 48 26 PM](https://github.com/Dokploy/dokploy/assets/47042324/6797c374-7874-4472-87fd-39c41e8d1c27)